index.js

```diff
@@ -187,8 +187,9 @@ Console.prototype.start = function ConsoleStart() {
         self.browserConnected = true;
         self.emit('logJSON', '');
     });
-    this.utWss.start(this.httpServer.listener);
     this.utWss.registerPath('/status');
+    this.utWss.ignorePath('/socket.io/');

```

Comment:
Where does this method 'ignorePath' come from?

Reply:
Sorry, please read the new description. I have removed this code.